src/utils/fs.rs

```diff
@@ -16,6 +16,27 @@ pub fn file_to_string<P: AsRef<Path>>(path: P) -> Result<String> {
     Ok(content)
 }
 
+/// Changes path to use standard UNIX/URL separators
+pub fn normalize_path(path: &str) -> String {
+    use std::path::is_separator;
+    path.chars()
+        .map(|ch| if is_separator(ch) { '/' } else { ch })
+        .collect::<String>()
+}
+
+/// Write the given data to a file, creating it first if necessary
+pub fn write_file<P: AsRef<Path>>(
```

Comment:
**Thank you** for removing this from the HTML renderer! It actually started life as a helper method on `MDBook` and got moved out because it coupled the entire HTML renderer to `MDBook`'s internals. At the time I was refactoring a different part of the system, so didn't want to go off on a tangent by also fixing `write_file()`.